CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # bedrock-vue-vc ChangeLog
 
+## 4.1.0 - 2024-mm-dd
```

Comment:
I think any changes to an existing UI look and feel is considered a breaking change in a UI package. Only adding new optional-to-use components (minor release) or fixing bugs without changing UI, unless part of the bug fix, (patch release) are non-major / non-breaking changes.

```suggestion
## 5.0.0 - 2024-mm-dd
```